content/en/docs/tasks/configure-pod-container/configure-gmsa.md

```diff
@@ -6,24 +6,20 @@ weight: 20
 
 {{% capture overview %}}
 
-{{< feature-state for_k8s_version="v1.14" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.16" state="beta" >}}
 
-This page shows how to configure [Group Managed Service Accounts](https://docs.microsoft.com/en-us/windows-server/security/group-managed-service-accounts/group-managed-service-accounts-overview) (GMSA) for pods and containers that will run on Windows nodes. Group Managed Service Accounts are a specific type of Active Directory account that provides automatic password management, simplified service principal name (SPN) management, and the ability to delegate the management to other administrators across multiple servers.
+This page shows how to configure [Group Managed Service Accounts](https://docs.microsoft.com/en-us/windows-server/security/group-managed-service-accounts/group-managed-service-accounts-overview) (GMSA) for Pods and containers that will run on Windows nodes. Group Managed Service Accounts are a specific type of Active Directory account that provides automatic password management, simplified service principal name (SPN) management, and the ability to delegate the management to other administrators across multiple servers.
 
-In Kubernetes, GMSA credential specs are configured at a Kubernetes cluster-wide scope as custom resources. Windows pods, as well as individual containers within a pod, can be configured to use a GMSA for domain based functions (e.g. Kerberos authentication) when interacting with other Windows services. As of v1.14, the only container runtime interface that supports GMSA for Windows workloads is Dockershim. Implementation of GMSA through CRI and other runtimes is planned for the future.
-
-{{< note >}}
-Currently this feature is in alpha state. While the overall goals and functionality will not change, the way in which the GMSA credspec references are specified in pod specs may change from annotations to API fields. Please take this into consideration when testing or adopting this feature.
-{{< /note >}}
+In Kubernetes, GMSA credential specs are configured at a Kubernetes cluster-wide scope as Custom Resources. Windows Pods, as well as individual containers within a Pod, can be configured to use a GMSA for domain based functions (e.g. Kerberos authentication) when interacting with other Windows services. As of v1.16, the Docker runtime supports GMSA for Windows workloads.
```

Comment:
@ddebroy , I read through the page again. A few more nits:
- line 13: To clarify, the last sentence is: As of Kubernetes v1.16, the Docker runtime ...? Or are you referring to the Docker version here?
- line 22: change the docs link to the feature gates page (minus the backticks):
`See [Feature Gates](/docs/reference/command-line-tools-reference/feature-gates/) for an explanation of enabling or disabling feature gates.`
- line 25: change the docs link to the CRD page:
`A [CustomResourceDefinition](/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/)`
- Try to capitalize the word Pod throughout the page. There are just a few places remaining.
- For consistency, change the instances of "use" verb to `use` verb?
- line 204: Should be changed to: If you are having difficulties getting GMSA to work in your environment, there are a few troubleshooting steps you can take:
  OR: If you are having difficulties getting GMSA to work in your environment, you can follow these steps to troubleshoot your setup:

Reply:
Line 13, the last sentence is indeed `As of Kubernetes v1.16, the Docker runtime`. Not referring to Docker versions.

Addressed the other nits. Thanks for reviewing!